subiquitycore/ui/form.py

```diff
@@ -160,6 +139,10 @@ def __init__(self, field, form, widget):
         if isinstance(widget, WantsToKnowFormField):
             widget.set_bound_form_field(self)
 
+    def is_in_error(self) -> bool:
+        """ Tells whether this field is in error. """
```

Comment:
I guess maybe this should check self._enabled too? Not sure.

Reply:
Adding this test crossed my mind multiple times.
I think generally speaking we never want errors in disabled fields to be considered meaningful. Or at least I can't think of any scenario where we would want that. So it makes sense to me to return False is the element is disabled.

The main reason why I didn't add this test is because I was afraid it could break something. For some reason, this test was not in place before (disabled fields did affect the done button) and I couldn't figure out if it was on purpose. Probably not?

Reply:
Yeah maybe something for another branch. I have a hard time believing we want an invalid disabled field to disable the done button.